bfio/bfio.py

```diff
@@ -25,7 +27,10 @@ def start() -> str:
         """
 
         global JAR_VERSION
-        loci = get_loci()
+        scyjava.config.endpoints.append("ome:formats-gpl:6.7.0")
+        scyjava.start_jvm()
+        loci = jpype.JPackage("loci")
+        loci.common.DebugTools.setRootLevel("ERROR")
```

Comment:
This code also appears in the backends. Do we need it in both places?

Reply:
Thanks for pointing to this. Updated the code so that we only do it in one place in `bfio.py` :-)